### PR TITLE
Add onboarding checklist

### LINKED ISSUE #259: Front end onboarding checklist template
In order for new team members to get acquainted with cloud.gov, there should be a checklist and list of items to go through.

Acceptance criteria:
- [ ] This list can be found in the `CONTRIBUTING.md` document

In order for current team members to successfully onboard a new team member, there should be a list of items to complete.

- [ ] Join the `#cloud-gov-frontend` channel on Slack
- [ ] Ping @standup-bot for instructions on front end channel standup (or view them in channel history)
- [ ] Review the main [front end board](https://github.com/18F/cg-deck#boards?repos=39210774)
- [ ] Review the cloud.gov [delivery process](https://github.com/18F/cg-product/blob/master/DeliveryProcess.md) to understand how the agile boards and workflow works.
- [ ] Bookmark link to [design folder](https://drive.google.com/drive/u/1/folders/0BwLqM4Nicmq-bUt0NjRjclFMUEU)
- [ ] Review the primary cloud.gov sites: [the deck](https://console.cloud.gov/#/), [main landing page](https://cloud.gov/), and [documentation](https://docs.cloud.gov/).

#### If developing
- [ ] Set up [the landing page site](https://github.com/18F/cg-landing) and/or [the deck](https://github.com/18F/cg-deck/tree/staging-alpha) and/or [the docs site](https://github.com/18F/cg-docs) and/or [style](https://github.com/18F/cg-style) locally
- [ ] Setup cloudgov-style to be [linked to the other sites locally](https://github.com/18F/cg-style#development-and-contributing-setup).
- [ ] Have cloud.gov person send the cg-deck testing env vars through fugacious
- [ ] Review [deck contributing guide](https://github.com/18F/cg-deck/blob/staging-alpha/CONTRIBUTING.md) and [cloudgov-style standards](https://github.com/18F/cg-style/blob/master/documentation/frontend_standards.md)

#### For review
- Review the [design resource request document](https://docs.google.com/document/d/1s96VP6PB7fbc8g_GwgAZ1hCPmew-J35ZOJx772c1AZ4/edit) if you haven’t already to get a sense of your role on the project
- Review the [design principals](https://docs.google.com/spreadsheets/d/14Y3RKaLUt6RPX5w13iz7oaSCpojEQ-Wqjnd8Ie_VkCc/edit#gid=259774738) to get a sense of how the cloud.gov team feels about the product
- Review the [competitive analysis](https://docs.google.com/spreadsheets/u/1/d/194tGz75NFwFyH8jT7zY_n-O0I7N4yAGYtIG4hw9qPcQ/edit?usp=drive_web) to get a sense of our "competitors" and their console/deck.
- Review the [cloudgov-style styleguide](https://pages.18f.gov/cg-style/) to get a sense of the global cloud.gov visual style.
- Review the [US Web Design Standards](https://standards.usa.gov/) as cloudgov-style was built from it.
- Review the deck, current [prod](https://console.cloud.gov/#/) and react refactor [staging](https://console-staging.cloud.gov/#/)
- Review the cloud foundry [community UI](http://ui.apps.cloud.gov/), a UI that members of the cloud foundry community created and is being used as a basis for our own deck design.

### PR DESCRIPTION
**Pull request by msecret**
Fixes #259

### CODE REVIEW

**Comment by mogul** [comment]
So you're suggesting we use the `cg-deck` repository as the central location for the Liberator stream despite the other front-end/user-facing repos? I was thinking making an explicit `cg-liberator` repository might be less confusing, similar to https://github.com/18F/cg-atlas and https://github.com/18F/cg-agent-q

**Comment by jeremiak** [comment]
Is there a public list we can use to reference what all these roles are? Its hard to hold them all in my head. I am worried that if I'm confused having repos named in that way will lead to more confusion for anybody not on the immediate team.

I'm fine with not having the front end on-boarding checklist be in `cg-deck`, in many ways it makes more sense for it to be in the `cg-style`. @mogul and @msecret what do you think about putting it there? Any sort of front end development is going to require `cloudgov-style` so that seems like a good place to put it

**Comment by msecret** [comment]
@jeremiak @mogul, it's in `cg-deck` now because we don't have a logical place for it and because we started using the agile board in `cg-deck`. I could see it making a little more sense on `cg-style` but because most of the contributor information is already on the deck and is probably the main body of work besides `cg-style` I'd rather keep it here until we've figured out a better place to keep it.

**Comment by mogul** [comment]
@jeremiak:
> Is there a public list we can use to reference what all these roles are? Its hard to hold them all in my head. I am worried that if I'm confused having repos named in that way will lead to more confusion for anybody not on the immediate team.

`cg-product` is intended to be the top-level cloud.gov repository, across all the sub-streams of activity. I'm just about to update [this file](https://github.com/18F/cg-product/blob/master/Onboarding.md#sub-teamsstreams-of-work) to reflect the new stream names/missions as described [here](https://docs.google.com/presentation/d/1nCcti3dXG9TVGW3OqaWtnf96oXX8U8SBTM_WePFO_dg/edit#slide=id.g111bc0346b_0_2232); sorry I've been slow on that. To your point, this info probably belongs right up front in the README.md, so will move that information there.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -113,6 +113,30 @@ Adding performance tracking and metrics is currently a TODO. Here are some items
 - What should performance budgets for decided metrics be? ie: faster then 1000 for speed index, faster then 1s for certain custom event, total request size below 2mb.
 - How should performance metrics and budgets be incorporated into workflow? Going over a budget requires re-implementation, or issue.
 - Any library added that's total file size is above 25kb should be evaluated for performance affect.
+- 
+
+## Onboarding checklist
+- [ ] Join the `#cloud-gov-frontend` channel on Slack
```
Comment (mogul) [inline CONTRIBUTING.md]: Should be `#cloud-gov-liberator`

**Comment by jeremiak** [comment]
Merging for now, as we can always move this content to a more suitable home when we find one.

**Pull request merged**